include/z64cutscene.h

```diff
@@ -22,6 +22,8 @@ typedef enum CutsceneState {
 } CutsceneState;
 
 typedef enum CutsceneCmd {
+    /* 0xFFFF */ CS_CMD_END_OF_SCRIPT = -1,
+    /* 0x0000 */ CS_CMD_NONE,
```

Comment:
```suggestion
```
I'm against creating a `CS_CMD_NONE` because `Cutscene_ProcessScript` does not implement reasonable behavior for a "NONE" command, and arguably having a "NONE" command does not fulfill any meaningful purpose within the code itself.

Reply:
I don't have a strong opinion either way.